semcore/d3-chart/src/a11y/hints.ts

```diff
@@ -188,4 +189,5 @@ export const makeDataSummarizationConfig = (
     notableDiff: undefined,
     ...(config?.movingAverage ?? {}),
   },
+  additionalFields: [...config.additionalFields]
```

Comment:
pls add optional chaining if config is not provided and ?? [] fallback if additionalFields is undefined